src/components/directory-content.js

```diff
@@ -734,83 +797,48 @@ const DirectoryContent = () => {
                     }
                 }}
             >
-                <ContentContextualMenu
-                    activeElement={activeElement}
-                    selectedElements={getSelectedChildren()}
-                    open={openContentMenu}
-                    openDialog={openDialog}
-                    setOpenDialog={setOpenDialog}
-                    onClose={handleCloseContentMenu}
-                    anchorReference="anchorPosition"
-                    anchorPosition={
-                        mousePosition.mouseY !== null &&
-                        mousePosition.mouseX !== null
-                            ? {
-                                  top: mousePosition.mouseY,
-                                  left: mousePosition.mouseX,
-                              }
-                            : undefined
-                    }
-                />
-
-                <DirectoryTreeContextualMenu
-                    directory={selectedDirectory}
-                    open={openDirectoryMenu}
-                    openDialog={openDialog}
-                    setOpenDialog={setOpenDialog}
-                    onClose={handleCloseDirectoryMenu}
-                    anchorReference="anchorPosition"
-                    anchorPosition={
-                        mousePosition.mouseY !== null &&
-                        mousePosition.mouseX !== null
-                            ? {
-                                  top: mousePosition.mouseY,
-                                  left: mousePosition.mouseX,
-                              }
-                            : undefined
-                    }
-                />
+                {openContentMenu && (
+                    <ContentContextualMenu
+                        activeElement={activeElement}
+                        selectedElements={getSelectedChildren()}
+                        open={openContentMenu}
+                        openDialog={openDialog}
```

Comment:
It's weird to keep "openContentMenu && " and "open={openContentMenu}".
Maybe for menus, we should keep it working with open={openContentMenu} as it will be often open and closed

Reply:
@sBouzols what do you think ?

Reply:
I don't find the `openContentMenu &&` part.
It's ok for me to render both ContentMenu and DirectoryMenu in the DirectoryContent and to control their showing by their open props. It's ok to use controlled framwork for those menu passing openDialog and setOpenDialog as well.
Will see in the future if it's a bad design...